godot-kotlin/godot-library/src/nativeCore/kotlin/godot/core/VariantArray.kt

```diff
@@ -1,33 +1,71 @@
 package godot.core
 
 import godot.gdnative.godot_array
-import kotlinx.cinterop.CValue
+import kotlinx.cinterop.*
 
-class VariantArray<T>(val value: CValue<godot_array>) : AbstractMutableList<T>() {
+class VariantArray<T> internal constructor(private val handle: CValue<godot_array>) : AbstractMutableList<T>() {
```

Comment:
I need handle to be accessible for api code generation.
See here: https://github.com/utopia-rise/godot-kotlin/blob/34d27b0532360c120424de5e885cad235a4b1d4f/buildSrc/src/main/kotlin/godot/codegen/ICall.kt#L103

Reply:
fixed in https://github.com/utopia-rise/godot-kotlin/pull/134/commits/2d61abf7b70d4738995c57343f0dffa617f5e463